README.md

```diff
@@ -19,13 +19,21 @@ And a larger data set
 
 ### Convert AnnData to Avro Load Format
 
-First convert from AnnData to the Avro format that will be used to load BigQuery:
+First convert from AnnData to the Avro format that will be used to load BigQuery. The `anndata_to_avro.py` script needs
+to know the appropriate values to use for cell and feature indexes. These can either be specified explicitly as command
+line arguments, or the script can be pointed to an existing dataset / project to figure those out itself.
+
+With explicit start values:
```

Comment:
I would flip the order of these, and make it clear the explicit start values is a "Note: if you need to explicitly…".  Your new way is much nicer!